Documentation/ceph-examples.md

```diff
@@ -45,7 +45,7 @@ Now that your operator is running, let's create your Ceph storage cluster. This
 that will influence how the operator configures the storage. It is important to understand the various ways to configure
 the cluster. These examples represent a very small set of the different ways to configure the storage.
 
-* `cluster.yaml`: This file contains common settings for a production storage cluster. Requires at least three nodes.
+* `cluster.yaml`: This file contains common settings for a production storage cluster. Requires at least three working nodes.
```

Comment:
"worker nodes" is better since it's an official name.

Reply:
Make sense. I updated the PR